pkg/certifier/scorecard/types.go

```diff
@@ -0,0 +1,40 @@
+//
+// Copyright 2023 The GUAC Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package scorecard
+
+import (
+	"context"
+
+	"github.com/guacsec/guac/pkg/assembler"
+	sc "github.com/ossf/scorecard/v4/pkg"
+)
+
+// scorecard is a struct that implements the Certifier interface.
+type scorecard struct {
+	scorecard Scorecard
+	ghToken   string
+	artifact  *assembler.ArtifactNode
+}
```

Comment:
nit: can probably be moved under `scorecard.go` since it is used in that context only.

Reply:
Moved the struct to `scorecard.go`